plasma_framework/contracts/src/exits/payment/controllers/PaymentStartInFlightExit.sol

```diff
@@ -361,6 +362,28 @@ library PaymentStartInFlightExit {
         return youngest;
     }
 
+    function getFurthestExitableTimestamp(StartExitData memory startExitData)
```

Comment:
the function `getYoungestInputUtxoPosition` and this `getFurthestExitableTimestamp` must be pointing to same input. I think currently we can potentially having a youngest position of an input and the furthest excitable timestamp to another input?

Reply:
It does not have to point to the same input - youngest input != furthest exitable timestamp.
When we were taking the exitable timestamp of the youngest input, we introduced a critical bug as described in #216. Now, my proposal is to use the last exitable timestamp (compute exitable timestamps of all inputs and choose one) and then still find the youngest input.

Reply:
I am a bit confused now (oops) I thought your proposal is to do what described in [this comment](https://github.com/omisego/plasma-contracts/issues/216#issuecomment-523399915) while we discussed offline.

I found a scenario current implementation seems wrong:

[valid_tx_1, valid_tx_2, invalid_tx, ...] -> [deposit_tx]....

Now operator IFE with input of `invalid_tx` and `deposit_tx`, the `furthestExitableTimestamp` would be of `invalid_tx` while the `youngest_utxo_pos` would be of `deposit_tx`. However, when we enqueue, we are only using the `txPos` instead of `utxoPos` which would be 0 for `deposit_tx` and make this IFE exiting with the same priority of `valid_tx_1` (since txPos would become 0). now `valid_tx_2` would have lower priority.....

Reply:
actually `tx pos` consists of `blknum` and `tx index`, which means that the priorities are still good.